content/en/docs/how-tos/add-jobs-to-testgrid.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Testgrid"
+date: 2020-10-28T11:14:39-04:00
+draft: false
+---
+
+We talk about two distinct components in this doc - testgrids and release-informing/blocking/broken jobs. 
```

Comment:
Did you mean to have this file in here? If not, let's remove it and merge the other, following up with the architects to modernize the content.

Reply:
Yeah my bad, removing it.